### DISCUSSION TITLE
Is the AviUtl Plugin supported?

### DISCUSSION THREAD

**Discussion by f11894** [HomeOfAviSynthPlusEvolution/L-SMASH-Works#8] [Q&A] [answered]
Do you mind if I report an issue about the AviUtl Plugin to this repository?

**Comment by realfinder** [comment, marked as answer]
Personally, I do not use AviUtl, and I think other members are same

I don't mind issue report about AviUtl, but I don't think it will get fixed by us so if you can do fixes you can do pull request or wait until some AviUtl user who can also do codes do pull request

also I don't remember that the [releases](https://github.com/HomeOfAviSynthPlusEvolution/L-SMASH-Works/releases) has other than avs and vs

**Reply by msg7086** [reply]
Same here. If it's an obvious bug that can be easily fixed without knowing much about AU, probably some of us can get it fixed. But don't expect too much from us.

You are free to file an issue though.

**Reply by f11894** [reply]
I got it,thank you.
